.ci/install_firecracker.sh

```diff
@@ -67,7 +67,7 @@ else
 	cat <<-EOF | sudo tee "$docker_configuration_file"
 	{
 	 "runtimes": {
-	  "kata": {
+	  "kata-runtime": {
```

Comment:
Why has this changed?

Reply:
@jodh-intel , if we leave it as `kata` all the tests are going to fail as they are looking for runtime `kata-runtime` that is why I am changing it.

Reply:
Oh - so this never worked? I understand now.

But this change should really be on a separate commit as it's not strictly related to the rest of the changes on this commit.